content/md/retailer-onboarder/first-steps/retailer-synchronization.md

```diff
@@ -16,21 +16,35 @@ You will find here all the necessary information related to synchronization:
 
 # Catalog structure
 
-During the supplier creation you will be able to link one or n families to a supplier.
+When you create a new supplier, you need to link families and catalog locales to a supplier.
 What does it mean and what is the impact?
```

Comment:
Note: you do not explain here how to link  families and catalog lcoales to a supplier. Maybe you could add a link somewhere on this sentence to the documentation where you explain it?